testing/bdd/features/timing/delay.feature

```diff
@@ -68,13 +68,16 @@ Feature: Delay of a document sequence
     | 00:00:05   |              |           |          |          |          | 00:00:10 | 00:00:15           |                  | 00:00:15          |                 | 00:00:15        |               |            |          |          |
```

Comment:
Actually this is what was tested previously, but we changed the code with Zoltan last week to check computed times instead. So, should we also check `begin` and `end` attributes ?